.github/workflows/ci.yml

```diff
@@ -54,6 +52,12 @@ jobs:
                     # most recent versions
                     - php: 8.0
                       symfony_version: 5.3.*
+                    - php: 8.0
+                      symfony_version: 5.4.*
+                    - php: 8.0
+                      symfony_version: 6.0.*
+                    - php: 8.1
```

Comment:
Missing symfony version? :thinking:

Reply:
All supported Symfony versions are present in at least one test case.

For this one, I leave the default version of Composer.
Do you prefer I specify a version for this one ?

Reply:
No it's ok, I was wondering what the `symfony/framework-bundle:${{ matrix.symfony_version }}` would do without a value, but it seems to work fine :ok_hand:

Reply:
The `symfony/framework-bundle` update is conditional:

```yaml
            - name: Install Symfony ${{ matrix.symfony_version }}
              run: composer require "symfony/framework-bundle:${{ matrix.symfony_version }}" --no-update
              if: matrix.symfony_version != ''
```